GoMoney/Common/ActivityItemSource/GMActivityItemSource.swift

```diff
@@ -0,0 +1,35 @@
+import LinkPresentation
+
+class GMActivityItemSource: NSObject, UIActivityItemSource {
+    var title: String
+    var text: String
+    var image: UIImage?
+
+    init(title: String, text: String, image: UIImage? = nil) {
+        self.title = title
+        self.text = text
```

Comment:
missing set image

Reply:
removed
This is just working with text, not Image